#!usr/bin/env python3
PACKAGE = "follow_line_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("thresh", int_t, 0, "Thresh Value", 150, 0, 255)
gen.add("speed", double_t, 0, "Speed Value", 0.0, 0.1, 4.0)
gen.add("enable_drive", bool_t, 0, "Enable", False)

exit(gen.generate(PACKAGE, "follow_line", "FollowLine"))


